### ISSUE TITLE
Build accessibility audit report

### ISSUE THREAD

**Issue by BigBen-7** [PinSpace-Org/GistPin#281]
## Description
WCAG compliance metrics.

## Requirements
- Compliance score
- Violation list
- Color contrast results
- Keyboard coverage

## Files to Touch
- analytics/app/accessibility/page.tsx

**Complexity: 150 points**

**Comment by Danielodingz** [comment]
@Danielodingz has applied to work on this issue as part of the **Stellar Wave Program**'s 4th wave.


> Hey maintainer, please i would like to work on this issue. 


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/7bd33ebc-ab2c-4403-81b3-8c5011779926) or assign @Danielodingz to this issue.

**Comment by AbdulmujibOladayo** [comment]
@AbdulmujibOladayo has applied to work on this issue as part of the **Stellar Wave Program**'s 4th wave.


> Hi maintainer. I am a software engineer whom have worked on this project before in earlier wave. I have a proven track record and already understand the codebase and how to implement this issue without obstacles.


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/7bd33ebc-ab2c-4403-81b3-8c5011779926) or assign @AbdulmujibOladayo to this issue.
